docs/FAQ.md

```diff
@@ -131,3 +131,90 @@ There is planned support for:
+
+If the transaction isn't accepted at all by Core, and you can't see the transaction on a [block explorer](https://www.blocktrail.com/BTC/), 
+
+## I'm tired of Armory! I want out!!!
+If all else fails and you don't want to use Armory any more, you have a couple of options.
+
+- If you can access your coins and don't mind a potential loss of privacy, you can just have all of the coins sent to an address controlled by an alternate wallet you wish to use.
+- If you can't access your coins, you can export the private keys into a wallet you trust. (FINISH)
+
+## Will Armory support Lightning?
+To be determined. (Caveat: I don't know goatpig's feelings, so the following is my opinion only for now.) Lightning is fascinating technology. Lightning also functions differently compared to Core. It may be possible for Armory to work with a Lightning node in a manner similar to a Core node. This way, Armory doesn't have to be redundant and re-implement various features required by the Lightning network.
+
+In theory, Armory can already supply the coins for the initial channel opening, and an address for the channel closing. The coins may need to come from a SegWit wallet (CONFIRM) but channel openings & closing are simple Bitcoin transactions. If the Lightning nodes can talk to Armory, it may be possible to provide further functionality. If not, support in Armory is highly doubtful. There's also an open question regarding whether or not Armory can or should monitor the network for events that mandate a channel closing.
+
+## Will Armory support Bitcoin Cash?
+Sort of. Armory does have the Cash signing algorithm and is, as of the beginning of 2018, able to send and receive coins on the Bitcoin Cash network. However, goatpig, the maintainer of Armory, has made it clear that his support of Cash on Armory is essentially limited to allowing users to move their coins to another wallet ASAP. Anybody expecting to use Armory with Cash long-term runs the risk of being unable to access their coins in the event of a fork (soft or hard) on the Cash network.
+
+## How can I use Armory and Bitcoin Cash?
+[This forum thread](https://bitcointalk.org/index.php?topic=2070058.0) started by goatpig explains how to safely move your coins. If you've already used the UTXOs on the Core network (i.e., spent all the coins or moved all the coins to other addresses), you don't have to worry about [replay attacks](https://bitcoin.stackexchange.com/a/61213). If you haven't, follow goatpig's directions for ensuring that you're not susceptible to replay attacks.
```

Comment:
> on the Core network

This should say "Bitcoin network". Calling it "Core" can be confusing and feeds into some FUD.